### PR TITLE
Change default logback level to warn in atlas

### PR DESCRIPTION
**Pull request by RemiDesgrange**

### CODE REVIEW

File: atlas/logback/logback.xml [hunk 1]
```diff
@@ -3,7 +3,7 @@
 
     <appender name="standardOut" class="ch.qos.logback.core.ConsoleAppender">
         <encoder>
-            <Pattern>%d{HH:mm:ss.SSS} [%thread] %-5level %logger{36} - %msg%n</Pattern>
+            <Pattern>JEANMICH %d{HH:mm:ss.SSS} [%thread] %-5level %logger{36} - %msg%n</Pattern>
```
Comment (pmauduit) [inline atlas/logback/logback.xml]: ? ;)

Comment (RemiDesgrange) [reply atlas/logback/logback.xml]: Ok Done :-)

**Comment by fvanderbiest** [comment]
Thanks !
Base branch should be 19.04.

Indeed, we merge:
 * `19.04` into `master`
 * `19.04` into `docker-19.04`
 * `master` into `docker-master`

**Comment by fvanderbiest** [comment]
Please target 19.04 branch instead.

**Pull request merged**